src/oaklib/cli.py

```diff
@@ -3713,6 +3713,12 @@ def diff_via_mappings(
     show_default=True,
     help="Delimiter between columns in input and output",
 )
+@click.option(
+    "--comment",
```

Comment:
hmm, I prefer something more descriptive but I guess this is consistent with pandas

Reply:
Initially i thought of `--comment-indicator` but as you mentioned, went with the pandas param